client/app/helpers/format-date.coffee

```diff
@@ -1,10 +1,11 @@
 `import Ember from 'ember'`
 
-formatDate = (date, options) ->
+formatDate = (date, format) ->
```

Comment:
thanks @4thethrillofit ! the way I've test it was replacing the code with these changes and then try load the /profile page